src/Pos/Launcher/Param.hs

```diff
@@ -30,7 +34,17 @@ data BaseParams = BaseParams
     { bpLoggingParams   :: !LoggingParams  -- ^ Logger parameters
     } deriving (Show)
 
--- | Contains algorithm specific & storage parameters for Node.
+-- | Network parameters.
+data NetworkParams = NetworkParams
+    { npDiscovery :: !(Either (Set NodeId) KademliaParams)
+    -- ^ If this value is 'Left', then given peers will be used in static mode.
+    -- Otherwise kademlia with given params will be used.
+    , npTcpAddr   :: !TCP.TCPAddr
```

Comment:
So I am leaving `External TCP address of the node`, because I don't want to accidentally lie.